games/openttd/patch-build.diff

```diff
@@ -0,0 +1,36 @@
+diff --git config.lib config.lib
```

Comment:
This file needs to be in a `files/` subdirectory, or MacPorts ignores it and attempts to download it from `master_sites`. This causes the CI build to fail.